__tests__/app.test.js

```diff
@@ -142,12 +142,28 @@ describe("GET /api/articles/:article_id/comments", () => {
         expect(Array.isArray(body.comments)).toBe(true);
       });
   });
+  test("404: responds with appropriate message when given valid but non-existent id.", () => {
+    return request(app)
+      .get("/api/articles/99999/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Not Found");
+      });
+  });
+  test("400: Bad request, INVALID id", () => {
+    return request(app)
+      .get("/api/articles/invalid_id/comments")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
```

Comment:
There's one more test you can add - there is an article that exists but doesn't have any comments yet. It should return a 200 and an empty array rather than an error. You will have to create a manual check as to whether the article exists or not to get this to work properly 👍